node/src/gql/mutation/login-google.ts

```diff
@@ -79,7 +79,7 @@ export const loginGoogle = {
         throw new Error('failed to create user');
       }
       // Create a new mentor if creating a new user account
-      await MentorSchema.findOneAndUpdate(
+      const mentor = await MentorSchema.findOneAndUpdate(
```

Comment:
Instead of findOneAndUpdate a mentor here, can check if User has any mentors already and if they don't, then create a new mentor for the user.